src/hangar/mixins/datasetget.py

```diff
@@ -314,7 +313,7 @@ def get(self, arraysets, samples, *, except_missing=False):
 
             # Data Retrieval
             asetsSamplesData = []
-            for aset in arraysets:
+            for aset in columns:
                 aset_samples = []
                 for sample in samples:
                     try:
```

Comment:
`arr = aset.get(sample)` will not throw key error and return `default`. So even if the argument `except_missing` is `True`, `co['aset', 'wrongkey']` will not raise an error. I assume this line should be `aset[sample]`

Reply:
Forgot to mention that this issue is not related to this PR. Something we have missed in the past review and our test suite couldn't catch

Reply:
Yup! Making the change now and adding a test!